auth_backends/backchannel_logout.py

```diff
@@ -105,7 +105,7 @@ def backchannel_logout(self, *args, **kwargs):
                 from users.models import TunnistamoSession
                 try:
                     tunnistamo_session = TunnistamoSession.objects.get(pk=session_data['tunnistamo_session_id'])
-                    tunnistamo_session.end()
+                    tunnistamo_session.end(send_logout_to_apis=True, request=self.request)
```

Comment:
Looks like there are no tests that check that when the back-channel logout endpoint of Tunnistamo is called, does Tunnistamo also call the other services' back-channel logout endpoints at that point.

Reply:
True. I'll need to add a test for that.